get_nxpRdLib.sh

```diff
@@ -6,72 +6,38 @@ NC='\033[0m' # No Color
 prereq() {
   echo -e "[${blue}Installing prerequisites${NC}]"
   sudo apt-get update
-  sudo apt-get -y install libglib2.0 glib-networking-services libreadline6-dev libglib2.0-dev glib-networking-services python2.7-dev build-essential autoconf libtool
+  sudo apt-get -y install python2.7-dev build-essential cmake
```

Comment:
Can you remove python2.7-dev from this line?  I've been meaning to do this anyway.